File: CN/q4.c
#include<stdio.h>

struct num {
    char c1;
    char c2[2];
    char c3;
};

int main() {
    struct num n = {1+'0', 23+'0', 4+'0'};
    printf("%c%s%c", n.c1, n.c2, n.c3);
    return 0;
}

File: DA Lab/Day 1/code/q4.r
num <- as.integer(readline(prompt = "Enter a 3-digit number: "))
digit1 <- num %/% 100
digit2 <- (num %/% 10) %% 10
digit3 <- num %% 10
sum_of_digits <- digit1 + digit2 + digit3
print(paste("Sum of the digits:", sum_of_digits))
